documents/charter.md

```diff
@@ -41,6 +41,13 @@ The mechanism must be scalable and implementable by Internet infrastructure as i
 1. Security.
 The mechanism will have the ability to invoke security mechanisms that provide confidentiality, integrity, and authenticity of the communication. The working group will consider the value and implications of different confidentiality modes of the communication.
 
+The following topics are out of scope for SCONEPRO:
```

Comment:
I will perhaps rephrase it as below 

- Support for streaming video flows carried in other transports and substrates than QUIC.
- Support for other media types that would require awareness of additional network attributes beyond the attributes applicable to ABR video.
- Support for general purpose network attributes. If additional network attributes are identified, the working group will request recharter to add them to SCONEPRO.
- Support for congestion signaling from the network. SCONEPRO should not be treated as mechanism to replace congestion control or rate adaptation.

Reply:
I very much like and agree with that last bullet. It should dispel many potential misunderstandings.

Reply:
@zaheduzzaman - I've made this change, and don't object to merging the edited list, but I do wonder if streaming video flows carried in H3 transport is **clearly** in scope (or out of scope) for this work. If it's clearly in scope, that's cool, but if anyone else is uncertain, we should talk before the BOF!